proseco/guide.py

```diff
@@ -635,7 +642,7 @@ def get_ax_range(r, extent):
     # Define the 1/2 pixel region as half the 8x8 plus dither
     row_extent = np.ceil(4 + dither.row)
     col_extent = np.ceil(4 + dither.col)
-    for idx, cand in enumerate(cand_stars):
+    for cand in cand_stars:
```

Comment:
This is not strictly required for this fix, but idx is redefined below and wasn't used in this context, so that was confusing.

Reply:
No problem, not worth a separate PR for tiny cleanup like this.